index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@400;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Zilla+Slab:ital,wght@0,300;0,400;0,600;0,700;1,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Tinos&display=swap" rel="stylesheet">
+    <title>apple.com</title>
+</head>
+<body>
+    <main class="main-box">
+        <div class="upper-section">
+            <div class="top-box">
```

Comment:
- [ ] The background should fill the whole page (without this white space in the borders). To achieve this, you can remove the margins of these elements and from their parents.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88217065-63072600-cc34-11ea-8888-ee30bd99d7e2.png) | ![image](https://user-images.githubusercontent.com/61479648/88217093-6ac6ca80-cc34-11ea-9bbb-1f314e224cc9.png) |